code/mod_table/mod_table.cpp

```diff
@@ -382,19 +382,19 @@ void parse_mod_table(const char *filename)
 
 		if (optional_string("$Generic Pain Flash Factor:")) {
 			stuff_float(&Generic_pain_flash_factor);
-			if (Generic_pain_flash_factor != 1.0f)
```

Comment:
I just noticed that this isn't being compared to 0, it's being compared to 1.  So it will need `fl_equal` instead.

Reply:
Good catch, just changed.